### PR TITLE
Add specs for templates

### LINKED ISSUE #14: Add specs for templates

### PR DESCRIPTION
**Pull request by tadatuta**
Closes #14

### CODE REVIEW

File: blocks/suggest/suggest.bemhtml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 block('suggest')(
-    def().match(function() { return !this._suggest; })(function() {
```
Comment (tadatuta) [inline blocks/suggest/suggest.bemhtml:2]: seems like useless?

File: blocks/suggest/suggest.bemhtml [hunk 2]
```diff
@@ -21,6 +21,7 @@ block('suggest')(
                     target : 'anchor',
                     autoclosable : true
                 },
+                attrs: { 'aria-hidden': "true" },
```
Comment (tadatuta) [inline blocks/suggest/suggest.bemhtml]: the bug is actually somewhere between `bem-xjst` and `bem-components` but I added it here to make spec pass until we fix it elsewhere

Comment (tadatuta) [inline blocks/suggest/suggest.bemhtml]: https://github.com/bem/bem-components/issues/1745

File: .enb/make.js [hunk 3]
```diff
@@ -9,14 +9,17 @@ var fs = require('fs'),
...
     bemhtml = require('enb-bemxjst/techs/bemhtml'),
     bh = require('enb-bh/techs/bh-commonjs'),
+    bhBundle = require('enb-bh/techs/bh-bundle'),
     bemjsonToHtml = require('enb-bemxjst/techs/bemjson-to-html'),
     //bemjsonToHtml = require('enb-bh/techs/bemjson-to-html'),
     browserJs = require('enb-js/techs/browser-js');
 
-const TESTS_PATH_RE = /(\w[a-z0-2_-]+)\.(tests|examples)\/(\w+)\.bemjson\.js$/;
+const TESTS_PATH_RE = /(\w[a-z0-2_-]+)\.(tests|examples)\/(\w+)\.bemjson\.js$/,
+      TMPL_ENGINE = process.env.TMPL_ENGINE || 'bemhtml';
```
Comment (narqo) [inline .enb/make.js]: 4 spaces would be enough for me.

File: blocks/suggest/suggest.bemhtml (hunk 2 above)
Comment (narqo) [inline blocks/suggest/suggest.bemhtml]: Could you add this comment to the code, so it would be clear that is's a hack, not an over-engineering?

**Comment by tadatuta** [comment]
:up:

**Pull request merged**